fuzzer/README.md

```diff
@@ -0,0 +1,75 @@
+# Build
+
+To build the `Fuzzy Wuzzy` executable, simply run `go build`. To build the module images, run `bash build_images.sh`.
+
+# Usage 
+
+Once built, `Fuzzy Wuzzy` is invoked like any other executable. 
+By default, `Go` places the executable in the same folder from which
+`go build` was invoked, with the executable taking the name of the the containing folder.
```

Comment:
How about adding a concrete example of what to run in bash after building?

Reply:
Good call